Dockerfile.buildtests

```diff
@@ -30,6 +30,11 @@ RUN ./configure && make && cp -f slirp4netns /
 
 FROM buildtest-centos76-common AS buildtest-centos76-static
 RUN yum install -y glibc-static glib2-static
+# FIXME: not sure how to install libseccomp-static via yum https://blog.51cto.com/bingdian/2347093
```

Comment:
@giuseppe @rhatdan PTAL? Is there any reason to exclude libseccomp-static from yum repo?

Reply:
you could probably use it from the `https://cbs.centos.org/repos/virt7-container-common-candidate/x86_64/os/` repository:

```
# yum-config-manager --add-repo=https://cbs.centos.org/repos/virt7-container-common-candidate/x86_64/os/ && \
 yum install --nogpgcheck -y libseccomp-static
```
```

Reply:
thanks